templates/jenkins-sysvinit.sh.j2

```diff
@@ -33,7 +35,7 @@ case "$1" in
     stop
     ;;
   status)
-    [ -f {{ jenkins_pid_file }} ]
+    status_of_proc -p {{ jenkins_pid_file }} /usr/bin/java java && exit 0 || exit $?
```

Comment:
FWIW, I more or less copied the usage from `/etc/init.d/ssh` in the `ubuntu` Docker image.